### DISCUSSION TITLE
[RELEASE] Pub

### DISCUSSION THREAD

**Discussion by Paindeluxe** [nzp-team/nzportable#283] [Map Releases]
This is a small map with next to no space to train the zombies.

_How long will you last?_

This most likely runs pretty well on the PSP, as I have tested it near the end of Development.

Story of this map

> You are a War Veteran at the Pub.
> You had a few too many drinks.
> You start to hallucinate.

# Preview
![Pub lscreen](https://user-images.githubusercontent.com/104042185/229524253-be2d3cfe-a0d7-465e-bc3e-bcff7eff6831.png)
![Pub Menü](https://user-images.githubusercontent.com/104042185/229524300-0eac12f7-a5b0-4296-812d-f9de89c89637.png)

If you want to edit the map or see how I made this, click [THIS](https://github.com/nzp-team/nzportable/files/13624180/Source.zip) to download the uncompiled version of the map. If not, click the [DOWNLOAD](https://github.com/nzp-team/nzportable/files/13624163/Pub.zip) button down below

# [DOWNLOAD](https://github.com/nzp-team/nzportable/files/13624163/Pub.zip)

**Comment by MotoLegacy** [comment]
Hey there, to be eligible for the "Meets Install Standard" label, please separate the map source files into a separate download link, appropriately named "Source" or something, instead of "DOWNLOAD".

**Reply by Paindeluxe** [reply]
I have fixed it. better late than never!
